src/webinars/2023/industrial-data-node-red.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Connect, Integrate, Visual Industrial Production Metrics with Node-RED"
+subtitle: Join FlowForge's Steve McLaughlin as he showcases how easy it is to use Node-RED to visualize popular production metrics using Node-RED.
+image: /images/webinars/webinar-intro-ff.jpg
+date: 2023-04-27
+time: 17:00 CEST (11:00 ET) 
+duration: 60
+hosts: ["steve-mclaughlin"]
+hubspot:
+    formId: b4ec9aa1-8648-47b3-b908-ec8597e8735d
+---
+
+**Node-RED makes it very easy to connect industrial equipment so that data can be visualized to understand common production metrics like OEE, performance, cycle times etc. **
+
+<!--more-->
+
+In this webinar, attendees will learn how to create a Node-RED flow to:
+- Connect to an existing PLC using Modbus
+- Integrate data from an external source
+- Create a dashboard to visualize common production metrics
+- Filter the data to report by exception and store this data in a [InfluxDB] database.
```

Comment:
Must we use Influx? 
Is it your thoughts and intention we will be displaying the Time Series data in Graphana?

My personal thoughts were heading to to MS-SQL (as it is _very corporate_) OR PostGRES and have PDF reports generated from the data.

Reply:
MS-SQL if fine. I will update.